tests/resources/sound_datasets/esc50_download/README.md

```diff
@@ -0,0 +1,189 @@
+## ESC-50: Dataset for Environmental Sound Classification
```

Comment:
I'm not sure why this file is here either?

Reply:
Same as above issue on ESC-50.